### PR TITLE
fix: Return raw query and actual reason phrase from server response

### PR DESCRIPTION
**Pull request by conor-ward**
## Description
This PR includes:

- Updating the groupId
- Fix for returning Raw query when logging requests
- Fix for preserving case of reason phrase that is actually returned from server

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -69,39 +69,4 @@
         <connection>scm:git:git@github.com:zalando/logbook.git</connection>
         <developerConnection>scm:git:git@github.com:zalando/logbook.git</developerConnection>
     </scm>
-    <profiles>
```
Comment (conor-ward) [inline pom.xml:72]: I removed this but maybe the jars should be made available publicly. Was initially planning to deploy to our own nexus

Comment (PatrykDagielSmartbear) [reply pom.xml:72]: Just a small question - for us that would be: https://nexus.tools.ops.smartbear.io/nexus/#welcome ?
I could find information about few nexus instances - but some seem to be just for particular projects (for example just for TM4J).

**Review by PatrykDagielSmartbear — APPROVED** [review]
Looks OK for me

File: pom.xml (hunk 1 above)
Comment (conor-ward) [reply pom.xml:72]: Yeah thats the URL, I was thinking of using this https://github.com/marketplace/actions/generate-settings-xml-for-maven-builds for setting up a settings file in github actions for when we want to deploy to our own repository.

**Review by andersjaensson — APPROVED** [review]

**Pull request merged**